### PR TITLE
docs: updated docs

### PR DESCRIPTION
**Pull request by orion-cengage**
Updated pages without design guidelines and updated images

Issue: #1026

### CODE REVIEW

File: packages/react-magma-dom/src/components/Tooltip/index.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export interface TooltipProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
    * The content of the tooltip
    */
-  content: React.ReactNode;
+  content: any;
```
Comment (silvalaura) [inline packages/react-magma-dom/src/components/Tooltip/index.tsx:33]: I know this may make your local build unhappy, but let's revert this change

File: website/react-magma-docs/src/pages/design/indeterminate.mdx [hunk 2]
```diff
@@ -9,69 +9,13 @@ import { LeadParagraph } from '../../components/LeadParagraph';
 
 <PageContent componentName="indeterminate_checkbox" type="design">
 
-<LeadParagraph>
-  Selection controls allow the user to select options.
-</LeadParagraph>
-
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/design/indeterminate.mdx:15]: The spacing on the page gets a little odd when it doesn't start with a LeadParagraph. Fine to leave it out if it doesn't bother you, but would recommend keeping it for consistency.

File: website/react-magma-docs/src/pages/design/password-input.mdx [hunk 3]
```diff
@@ -24,277 +23,11 @@ import { Link } from 'gatsby';
...
 
-![ ](../../images/inputs/Input-anatomy.png)
+![ ](../../images/inputs/password-input-anatomy.png)
 
 1. Field label
-1. Input text
-1. Container
-1. Leading icon
-1. Trailing icon
+1. masked password
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/design/password-input.mdx:29]:
```suggestion
1. Masked password
```

File: website/react-magma-docs/src/pages/design/native-select.mdx [hunk 4]
```diff
@@ -12,8 +12,12 @@ import { LeadParagraph } from '../../components/LeadParagraph';
...
 <LeadParagraph>
   The Select component allows users to choose one or more options from a list.
-  It is used in forms for users to submit data.
 </LeadParagraph>
 
+## Usage
+
+The Native Select functionally works the same as the standard Select. The difference is that the Native Select lets the browser determine how the list of options is presented. This makes it the preferred component when the list of options is longer than 10 items. This will ensure the list is navigable and optimized for any size screen.
+
+![ ](../../images/inputs/native-select-example.png)
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/design/native-select.mdx:21]:
```suggestion
![](../../images/inputs/native-select-example.png)
```

**Review by ccedrone — APPROVED** [review]

File: website/react-magma-docs/src/pages/design/indeterminate.mdx (hunk 2 above)
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/indeterminate.mdx]: Fair point. I found some good text to put in place there.

File: website/react-magma-docs/src/pages/design/password-input.mdx (hunk 3 above)
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/password-input.mdx]: Fixed

Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/password-input.mdx]: Fixed

File: website/react-magma-docs/src/pages/design/password-input.mdx [hunk 5]
```diff
@@ -24,277 +23,11 @@ import { Link } from 'gatsby';
 
 ## Anatomy
 
-![ ](../../images/inputs/Input-anatomy.png)
+![ ](../../images/inputs/password-input-anatomy.png)
```
[the remark this answers is not in the document]
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/password-input.mdx]: Fixed

File: website/react-magma-docs/src/pages/design/combobox.mdx [hunk 6]
```diff
@@ -10,9 +10,19 @@ import { LeadParagraph } from '../../components/LeadParagraph';
...
+
+The list is designed to help the user arrive at a value, but the value does not necessarily have to come from that list. Don't think of it like a Select, but more of a text input with some suggestions.
+
 ![ ](../../images/inputs/combobox-example.png)
 
+## Multi Combobox
+
+The multi combobox allows the user to select more than one option from the list and add their own.
+
+![ ](../../images/inputs/multi-combobox-example.png)
```
[the remark this answers is not in the document]
Comment (orion-cengage) [reply website/react-magma-docs/src/pages/design/combobox.mdx]: Fixed

**Review by silvalaura — APPROVED** [review]
